tokio/src/runtime/time/mod.rs

```diff
@@ -99,7 +99,7 @@ impl AtomicOptionNonZeroU64 {
 /// either be canceled (dropped) or their associated entries will reach level
 /// zero and be notified.
 ///
-/// [paper]: http://www.cs.columbia.edu/~nahum/w6998/papers/ton97-timing-wheels.pdf
+/// [paper]: https://ieeexplore.ieee.org/document/650142
```

Comment:
We could use this link: 

https://web.archive.org/web/20240119063125/http://www.cs.columbia.edu/~nahum/w6998/papers/ton97-timing-wheels.pdf

Reply:
The original link loads for me. I don't think it's broken?

Reply:
Hmm, I tried it earlier today and it didn't work, but now it works. Seems flaky.

Reply:
I'm not 100% sure if this is the entire issue but I tried loading the URL in Firefox which works fine but when I try to load it in Safari it returns a File Not Found error page. Safari tries to force HTTPS which then fails to load the file while Firefox is fine loading the HTTP only version. So perhaps the browser being used is the issue?

Reply:
Yes, it worked well in Firefox, but it redirected to HTTPS in the ARC browser, so it didn't work.

Please let me know if you want me to change it to a HTTPS link.